core/src/main/java/io/camunda/migrator/impl/logging/RuntimeValidatorLogs.java

```diff
@@ -31,6 +31,7 @@ public class RuntimeValidatorLogs {
   public static final String FLOW_NODE_NOT_EXISTS_ERROR = "Flow node with id [%s] doesn't exist in the equivalent deployed C8 model.";
   public static final String NO_C8_DEPLOYMENT_ERROR = "No C8 deployment found for process ID [%s] required for instance with legacyID [%s].";
   public static final String FAILED_TO_PARSE_BPMN_MODEL = "Failed to parse BPMN model from XML";
+  public static final String CALL_ACTIVITY_LEGACY_ID_ERROR = "Found call activity with propagateAllParentVariables=false for flow node with id [%s] in C8 process. This is not supported by the migrator unless there is an explicit mapping for the legacyId variable, as it would lead to orphaned sub-process instances.";
```

Comment:
❓ So to clarify, all this needs to be applied to the C8 process, right? Not the C7 one

Reply:
Yes, in C8 it can be configures what variables from the call activity are shared in a subprocess. This can be set via `propagateAllParentVariables` or the `io mapping`. This does not exist in C7.